go/tasks/plugins/k8s/kfoperators/pytorch/pytorch.go

```diff
@@ -82,8 +83,23 @@ func (pytorchOperatorResourceHandler) BuildResource(ctx context.Context, taskCtx
 	}
 
 	workers := pytorchTaskExtraArgs.GetWorkers()
+	minReplicas := pytorchTaskExtraArgs.GetMinReplicas()
+	maxReplicas := pytorchTaskExtraArgs.GetMaxReplicas()
+	maxRestarts := pytorchTaskExtraArgs.GetMaxRestarts()
+	rdzvBackend := pytorchTaskExtraArgs.GetRDZVBackend()
+	nProcPerNode := pytorchTaskExtraArgs.GetNProcPerNode()
+
+	fmt.Printf("minReplicas")
+	fmt.Println(minReplicas)
 
 	jobSpec := kubeflowv1.PyTorchJobSpec{
+		ElasticPolicy: &kubeflowv1.ElasticPolicy{
```

Comment:
Of course an if statement whether the user configured elastic training is required.